extensions/wikia/ARecoveryEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointCSDelivery.php

```diff
@@ -1,28 +1,13 @@
 <?php
 class ResourceLoaderAdEngineSourcePointCSDelivery extends ResourceLoaderAdEngineSourcePointBase {
-	const TTL_SCRIPTS = 86400;   // one day for fresh scripts from SourcePoint
-	const TTL_GRACE = 3600; // one hour for old scripts (served if we fail to fetch fresh scripts)
 	const CACHE_BUSTER = 22;     // increase this any time the local files change
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)